src/Chainweb/Mempool/InMem.hs

```diff
@@ -260,16 +267,28 @@ getBlockInMem :: forall t .
 getBlockInMem cfg lock txValidate bheight phash size0 = do
     withMVar lock $ \mdata -> do
         !psq0 <- readIORef $ _inmemPending mdata
+        -- TODO: temporary: delete this in preference for Emmanuel's TTL branch
```

Comment:
Can this be dealt with yet?

Reply:
no

Reply:
...I expect this one to get in first at this rate

Reply:
... the time check is being done elsewhere and we are only using 'now' to clean out badmap -- i think this is ok